src/Magento/FunctionalTestingFramework/Util/Logger/MftfLogger.php

```diff
@@ -86,13 +81,10 @@ public function criticalFailure($message, array $context = [], $verbose = false)
     public function notification($message, array $context = [], $verbose = false)
     {
         $message = "NOTICE: " . $message;
-        // print during test generation
-        if ($this->phase === MftfApplicationConfig::GENERATION_PHASE && $verbose) {
+        // Suppress print during unit testing
+        if ($this->phase !== MftfApplicationConfig::UNIT_TEST_PHASE && $verbose) {
```

Comment:
Can we safely revert the code recently done for trimming the log file size?

Reply:
Um, we could. There is some code left behind from that PR like use of `$this->phase`. I figured, updating code would be easier, given that we'll need to delete history etc. Let me know otherwise.

Reply:
I am not clear what you mean. This change results in printing/logging notices, doesn't it?

Reply:
I would prefer keep the original behavior: do not print/log notices during runtime. One possible solution might be to use $context. If it's metadata deprecation notice, print/log it, else skip the notices

Reply:
fixed. Now using $context to print/log deprecation notices during runtime.